lib/api/routes.js

```diff
@@ -65,6 +65,8 @@ module.exports = function routes(parsoidConfig, processLogger, parsoidOptions, p
 		res.locals.oldid = req.params.revision || null;
 		// Fandom change
 		res.locals.mobileve = req.query.mobileve && req.query.mobileve !== 'false' ? '1' : '';
+		// SITE-940 Remove after full fandom desktop rollout
+		res.locals.fandomdesktop = req.query.fandomdesktop && req.query.fandomdesktop !== 'false' ? '1' : '';
```

Comment:
Can we instead enable Parsoid's native gallery handling when FD is available?

Reply:
Would it affect our plans to enhance galleries with a slider mode on FD?

Reply:
We don't change the backend rendering right?